### PR TITLE
Create Autonomous RC Car platform

### PR DESCRIPTION
**Pull request by Andriy-Malyshchak**
## Describe your project in 1-3 sentences

I am building a self-driving car on a 1/16 scale RC version.

Username AndriyM

## Checklist before submitting proposal:

- [ yes] I am a current high school student (or younger).
- [ yes] I have submitted an RSVP at <https://hackclub.com/winter>
- [yes ] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [yes ] I have self-reviewed my project proposal.

### CODE REVIEW

File: Autonomous RC Car platform [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Nvidia JetsonNano   | https://www.amazon.com/NVIDIA-Jetson-Nano-Developer  | $157  |
+| Wifi Card | https://www.amazon.com/Edimax-EW  | $9 |
+Camera | https://www.amazon.com/NVIDIA-Jetson-Development-Camera  | $10 |
+Servo Driver | https://www.amazon.com/Dorhea-PCA9685  | $9 |
+Antennas | https://store.donkeycar.com/products/2x-molex-wifi  | $7 |
+RC Car |  https://www.exhobby.com/products/short-course-30mph | $60 |
```
Comment (bellesea) [inline Autonomous RC Car platform:40]: did you meant to link this? https://www.exhobby.com/products/short-course-30mph-high-speed-truck-with-shock-absorber-system-and-water-splash-proof-structure

Comment (bellesea) [inline Autonomous RC Car platform:40]: did you meant to link this? https://www.amazon.com/Dorhea-PCA9685-Interface-Controller-Raspberry/dp/B07RMTN4NZ

**Review by bellesea — CHANGES_REQUESTED** [review]
@andriy-malyshchak this is such a cool idea!!! absolutely love it hehe :) 

i'm wondering, how do you know that your RC car is compatible with the servo driver? also, what's the role of the antennas?

and i see that you're buying wires, do you already have a breadboard?

thank you sm! btw, could you please update your links? i think most of them are cut off...

File: Autonomous RC Car platform (hunk 1 above)
Comment (Andriy-Malyshchak) [reply Autonomous RC Car platform]: Yep, I'm not sure why the links cut off?

Comment (Andriy-Malyshchak) [reply Autonomous RC Car platform]: Yeah, that's the one :)!

File: andriy-malyshchak.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Nvidia JetsonNano   | https://www.amazon.com/NVIDIA-Jetson-Nano-Developer-945-13450-0000-100/dp/B084DSDDLT/ref=sr_1_3?crid=8AG2PWH8H3MR&keywords=nvidia+jetson+nano&qid=1671664292&s=electronics&sprefix=nvidia+jetson+nano%2Celectronics%2C103&sr=1-3  | $157  |
+| Wifi Card | https://www.amazon.com/Edimax-EW-7811Un-V2-Compatible-2-6-18-4-14/dp/B08F2ZNC6J/ref=sr_1_1?crid=6NNV79DQX5LM&keywords=Edimax+EW-7811Un&qid=1671743931&sprefix=edimax+ew-7811un%2Caps%2C85&sr=8-1
+  | $9 |
+Camera | https://www.amazon.com/NVIDIA-Jetson-Development-Camera-Application/dp/B0854CP54X/ref=sr_1_3?crid=GF1ETUWHGVMK&keywords=nvidia+jetson+camera&qid=1671674825&refinements=p_36%3A-2000&rnid=386442011&sprefix=nvidia+jetson+camer%2Caps%2C105&sr=8-3
+
```
Comment (bellesea) [inline andriy-malyshchak.md:39]:
```suggestion
```

**Review by bellesea — CHANGES_REQUESTED** [review]
@Andriy-Malyshchak thanks for pushing the changes! 

are you following/referenceing a guide of sorts? if so, could you please share the relevant link(s)?

and, how do you know that your RC car is compatible with the servo driver? also, what's the role of the antennas and the wifi card?

would also you be able to expand a little bit on how your autonomous car would work? for example, how does your car receive instructions?

and, what would success of this project look like?

thank you sm!

File: andriy-malyshchak.md (hunk 2 above)
Comment (Andriy-Malyshchak) [reply andriy-malyshchak.md:39]: Hi! Here's the link for the DonkeyCar website: http://docs.donkeycar.com/.
The RC car that I chose is one of the models that the website lists as compatible with the hardware. As I move through different iterations, I'll probably run into more alternatives(these are just tested). 
The antennas and Wifi card are used for communicating to the ground station. Wifi card enables connection, while antennas boost that connection.
The car is connected to a laptop while coding and operates on its own via the different sensors on the car. The camera feeds the image into the Nvidia Jetson Nano => into PCA servo controller => motor/steering.
Initial success will be if the car is able to follow a demarked loop that I'll probably draw with chalk on the ground(or something similar).

Comment (bellesea) [reply andriy-malyshchak.md:39]: > Hi! Here's the link for the DonkeyCar website: http://docs.donkeycar.com/. The RC car that I chose is one of the models that the website lists as compatible with the hardware. As I move through different iterations, I'll probably run into more alternatives(these are just tested). The antennas and Wifi card are used for communicating to the ground station. Wifi card enables connection, while antennas boost that connection. The car is connected to a laptop while coding and operates on its own via the different sensors on the car. The camera feeds the image into the Nvidia Jetson Nano => into PCA servo controller => motor/steering. Initial success will be if the car is able to follow a demarked loop that I'll probably draw with chalk on the ground(or something similar).

@Andriy-Malyshchak that sounds great! could you please add this to the file? maybe under ##plan? thank you sm :))

Comment (Andriy-Malyshchak) [reply andriy-malyshchak.md:41]: Added to the Plan

**Review by bellesea — APPROVED** [review]
@Andriy-Malyshchak thanks for your patience with an updated review (we were out for christmas break), and thank you for pushing the changes :) 

a fabulous member of @hackclub/bank-ops will be reaching out soon with next steps!

can't wait to see you work on this :)

**Review by MelSmith104 — COMMENTED** [review]
hey @Andriy-Malyshchak this project sounds like so much fun!! i used to love playing around with rc cars and a self driving one sounds like a really awesome project :) im dm'ing you on slack right now so that we can chat about next steps!

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**